package.json

```diff
@@ -4,19 +4,22 @@
   "description": "Assemble: schoolofwok.co.uk, sow-backoffice, sow-api",
   "main": "index.js",
   "dependencies": {
+    "async": "^1.5.2",
     "chalk": "^1.1.0",
     "config": "^1.14.0",
     "faker": "^3.0.0",
     "moment": "^2.10.3",
+    "mongojs": "^2.2.2",
     "nightwatch": "^0.7.6",
+    "ps-tree": "^1.0.1",
     "selenium-standalone": "^4.5.3"
   },
   "devDependencies": {},
   "scripts": {
     "start": "node start.js",
     "test": "node test.js",
-    "get-selenium": "selenium-standalone install --version=2.46.0",
-    "selenium": "selenium-standalone start --version=2.46.0",
+    "get-selenium": "selenium-standalone install --version=2.46.0 --drivers.chrome.version=2.21",
```

Comment:
@richsilv that chromedriver version problem was due to not specifying it on the install step, so the latest would always be grabbed. I've updated the package.json to install the same version as nightwatch expects now.